ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-stable-4.11.yaml

```diff
@@ -369,25 +484,36 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-gcp-ipi-cco-manual-workload-identity
+- as: gcp-ipi-sdn-f28
```

Comment:
The same comment as https://github.com/openshift/release/pull/51335#discussion_r1582049482

Reply:
ok, will also remove same profile from z stream upgrade job

Reply:
I think we only need to remove the y stream upgrade job, because for z stream upgrade job, it comes to 4.12, the default networkType is changed to `OVN`, so need some explicit SDN job to run z stream upgrade.